src/ray/raylet/node_manager.h

```diff
@@ -26,6 +25,13 @@ namespace ray {
 
 namespace raylet {
 
+using rpc::ActorTableData;
+using rpc::ClientTableData;
+using rpc::DriverTableData;
+using rpc::ErrorType;
+using rpc::HeartbeatBatchTableData;
+using rpc::HeartbeatTableData;
```

Comment:
is it possible to move these using inside the NodeManager class, or move to .cc file?

Reply:
C++ doesn't seem to allow putting `using` in class definition.